pkg/dclient/certificates.go

```diff
@@ -39,107 +39,18 @@ func (c *Client) InitTLSPemPair(configuration *rest.Config, fqdncn bool) (*tls.T
 	return tlsPair, nil
 }
 
-//generateTlsPemPair Issues TLS certificate for webhook server using given PEM private key
+//buildTLSPemPair Issues TLS certificate for webhook server using self-signed CA cert
 // Returns signed and approved TLS certificate in PEM format
-func (c *Client) generateTLSPemPair(props tls.TlsCertificateProps, fqdncn bool) (*tls.TlsPemPair, error) {
-	privateKey, err := tls.TLSGeneratePrivateKey()
+func (c *Client) buildTLSPemPair(props tls.TlsCertificateProps, fqdncn bool) (*tls.TlsPemPair, error) {
```

Comment:
change case to `buildTlsPemPair` to be consistent with other type and method names.